docker-compose/docker-compose.yaml

```diff
@@ -2,23 +2,57 @@ version: '2.4'
 services:
   # Description: Acts as a reverse proxy for all of the sourcegraph-frontend instances
   #
-  # Disk: none
+  # Disk: 1GB / persistent SSD
```

Comment:
Why does Caddy need a persistent disk?

Reply:
See https://caddyserver.com/docs/conventions#data-directory and https://caddyserver.com/docs/conventions#configuration-directory